web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-studio.adoc

```diff
@@ -0,0 +1,276 @@
+  <wsc:connection
+   wsdlLocation="http://ilt.mulesoft-training.com/essentials/delta?wsdl"
+   service="TicketServiceService"
+   port="TicketServicePort"/>
+</wsc:config>
+----
+
+
+[[configure-other-fields]]
+== Configure Consume Operation
+
+After you configure a global element for Web Service Consumer Connector, configure the Consume operation parameters, such as Operation, Message (Body, Headers, Attachments) and so on. Additionally, learn about the output of the Consume operation and metadata attributes used to dispatch messages.
+
+=== Parameters
+
+The Consume operation has two main parameters:
+
+* *Operation* +
+Defines which SOAP operation of the Web service to invoke. In design time, the parameter defines the input and output types for the Consume operation. The types will change depending on which operation you choose.
+
+* *Message* +
+A representation of the `SOAP:ENVELOP` composed of three optional parameters:
+** *Body* +
+The XML body to include in the SOAP message, with all the required parameters, or `null` if no parameters are required.
+** *Headers* +
+The XML headers to include in the SOAP message.
+** *Attachments* +
+The attachments to include in the SOAP request.
+
+To configure these parameters for the Consume operation, follow these steps: +
+
+. Select the name of the connector in the Studio canvas.
+. On the *General* section of the Consume operation define *Operation*, and in the *Message* section define *Body*, *Headers* and *Attachments*:
+
+.Configure Consume Operation
+image::web-service-consumer-configure-consume.png[Configure Consume Operation]
+
+In the *Configuration XML* tab, a basic configuration for the Consume operation looks as follows:
+[source,xml,linenums]
+----
+<wsc:consume config-ref="config" operation="addClients">
+    <wsc:message>
+        <wsc:body>#[payload]</wsc:body>
+    </wsc:message>
+</wsc:consume>
+----
+
+==== Body Parameter
+
+The `body` parameter is the main part of the SOAP message, and accepts embedded DataWeave scripts values so that you can construct the XML request without having a side effect on the message or having to use multiple components to create the request. The default value is `#[payload]` and expects that the incoming payload is the XML entity ready to ship to the service. +
+If the body is not a valid XML, or if the request cannot be created for some reason, you will get a `WSC:BAD_REQUEST` error. If you don't provide body content, the Web Service Consumer Connector attempts to generate one, and this only works for cases where no XML entity is expected in the body. +
+Additionally, some Web Services require that the body has the XML initial declaration declared at the top of it. This tag has the version and encoding information that identifies the document as being XML.
+
+The following XML example shows a DataWeave expression inside the `body` parameter, and how to force the Web Service Connector to dispatch any payload setting the XML initial declaration:
```

Comment:
The last part of the sentence is unclear. I think "that sets" instead of "setting" makes it clearer.